packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -103,7 +103,7 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
     };
 
     const verifyRemainingLimit = (expectedCost: number, remainingHbarsBefore: number, remainingHbarsAfter: number) => {
-      const delta = 0.02 * expectedCost;
+      const delta = 0.05 * expectedCost;
```

Comment:
**Note for Reviewers:** 
Bumped tolerance to fix flaky test

Reply:
Something like this could be a TEST_Config to allow tweaking without code changes

Reply:
Yeah I agree. Pushed in an update which adds TEST_TRANSACTION_RECORD_COST_TOLERANCE config variable which can be reused through out the test.